feat(daokit): update daokit framework with latest version

davd-gzl opened the pull request:
Add the latest updated version of [daokit repository](https://github.com/samouraiworld/gnodaokit) done this last months to the mono-repo.
It was accomplished by @n0izn0iz, @WaDadidou, @MikaelVallenet,  @Davphla

This pull request add:
- Upgradable DAO - @n0izn0iz 
- Extension system - @n0izn0iz 
- Better UI/UX - @WaDadidou 
- Overall maintenance + Reviews - @MikaelVallenet, @n0izn0iz
- Full comprehensive documentation - @Davphla 
- Set of examples (/r/samcrew/daodemo) - @Davphla

davd-gzl committed:
feat(daokit): update daokit framework with latest update

Gno2D2 commented:
#### 🛠 PR Checks Summary
All **Automated Checks** passed. ✅

##### Manual Checks (for Reviewers):
- [ ] **IGNORE** the bot requirements for this PR (force green CI check)


<details><summary>Read More</summary>

🤖 This bot helps streamline PR reviews by verifying automated checks and providing guidance for contributors and reviewers.

##### ✅ Automated Checks (for Contributors):
 🟢 Maintainers must be able to edit this pull request ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
 🟢 Pending initial approval by a review team member, or review from tech-staff


##### ☑️ Contributor Actions:
1. Fix any issues flagged by automated checks.
2. Follow the Contributor Checklist to ensure your PR is ready for review.
	- Add new tests, or document why they are unnecessary.
	- Provide clear examples/screenshots, if necessary.
	- Update documentation, if required.
	- Ensure no breaking changes, or include `BREAKING CHANGE` notes.
	- Link related issues/PRs, where applicable.

##### ☑️ Reviewer Actions:
1. Complete manual checks for the PR, including the guidelines and additional checks if applicable.

##### 📚 Resources:
- [Report a bug with the bot](https://www.github.com/gnolang/gno/issues/3238).
- [View the bot’s configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 And
    ├── 🟢 The base branch matches this pattern: ^master$
    └── 🟢 The pull request was created from a fork (head branch repo: davd-gzl/gno)

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>Pending initial approval by a review team member, or review from tech-staff</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 And
    ├── 🟢 The base branch matches this pattern: ^master$
    └── 🟢 Not (🔴 Pull request author is a member of the team: tech-staff)

```
### Then
```
🟢 Requirement satisfied
└── 🟢 If
    ├── 🟢 Condition
    │   └── 🟢 Or
    │       ├── 🔴 At least one of these user(s) reviewed the pull request: [davd-gzl jefft0 notJoon omarsy MikaelVallenet] (with state "APPROVED")
    │       ├── 🟢 At least 1 user(s) of the team tech-staff reviewed pull request
    │       └── 🔴 This pull request is a draft
    └── 🟢 Then
        └── 🟢 Not (🔴 This label is applied to pull request: review/triage-pending)

```
</blockquote></details>

</blockquote></details>


<details><summary><b>Manual Checks</b></summary><blockquote>

<details><summary>**IGNORE** the bot requirements for this PR (force green CI check)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Can be checked by

- Any user with comment edit permission

</blockquote></details>

</blockquote></details>

</blockquote></details>

</details>

davd-gzl committed:
fix(daokit): add missing daodemo directory

davd-gzl committed:
Merge remote-tracking branch 'origin/master' into feat/daokit-v2

# Conflicts:
#	examples/gno.land/p/samcrew/basedao/render.gno

davd-gzl committed:
fix(daokit): use versioned gno.land/p/nt/* import paths

All gno.land/p/nt/* packages require the /v0 suffix. The unversioned
paths (ufmt, seqid, avl, avl/pager, mux) do not exist as modules.
